application/Hand.py

```diff
@@ -1,4 +1,30 @@
-class Hand: 
-  def __init__(self, cards, startingPlayer):
-    hand = cards
-    player = startingPlayer
+import psycopg2
+import random
+import application.Card as Card
+
+# Probably better to get rid of the hand class once there is a better place for createHands()
+class Hand:
```

Comment:
This class here will have to be changed to use the flask database helper library as discussed in meeting